configure

```diff
@@ -107,7 +107,7 @@ my %platforminfo =
                   'usage' => [
                       "Use --macros=ANDROID_ABI=<ARCH> to specify the",
                       "target architecture.",
-                      "Use --macros=ANDROID_API_PATH=<API_PATH> to specifify",
+                      "Use --macros=ANDROID_API_PATH=<API_PATH> to specify",
```

Comment:
Two things:
- New make variable names like this should probably be lowercase to match other Android make ACE variables. `ANDROID_ABI` will still work for some time, but it has already been basically replaced with `android_abi` in ACE. I was going to change references when I updated OpenDDS to build using the NDK directly, but haven't gotten around to it yet.
- So I think we should split this into two different variables: `android_sdk` and `android_target_api`. It's probably unlikely, but theoretically a user might want to use Java and target a platform lower than 24, so doesn't care about the API 30 feature. That way we can check and disable `AndroidNetworkCallback.java` if the API doesn't support it. On top of that the user doesn't really have to know about the location of `android.jar` within the SDK.

This is separate and doesn't have to be done now, but it would have been helpful when getting this set up for the matrix to check that the `android.jar` actually exists and print an error if it doesn't. Otherwise it just has a bunch of "undefined" errors and it's not immediately clear what the issue is. It happened to me because I was downloading the minimum API platform instead of the targeted API platform, but could happen if the user a number of ways.